_pages/cfp.md

```diff
@@ -1,22 +1,79 @@
 ---
 layout: page
-title: "BSidesSF Call For Participation"
+title: "BSidesSF 2025 Call For Participation"
 ---
 
-BSidesSF 2024 CFP is Closed! Thanks to all who submitted.
+BSidesSF is soliciting submissions for the annual BSidesSF conference on April 
+26-26, 2025. Call for Participation is currently open for all tracks.
 
-We received a total of 334 reviewable submissions (up 59.8% from 209 in 2023), but could only accept 60. This means we had an unusually low acceptance rate of 17.9% (compared to 28.7% in 2023), despite 45% of the submissions receiving favorable reviews.
+Note: We **DO NOT** provide speaker or travel stipends at this time (except on a 
```

Comment:
We don't provide travel stipends to villages either. We provide limited reimbursement. Stipend is generally an amount given without restrictions. We just reimburse. So, maybe change this to "travel stipends/reimbursement"?

Reply:
Changing stipend to reimbursements, and will keep this in mind when updating the villages pages.